lib/db.ts

```diff
@@ -37,6 +37,26 @@ export async function createUser(
     .values({ name, phone, email, password: hash });
 }
 
+export async function addProfilePictureUrlToUser(
+  values: typeof Schemas.files1._.inferInsert,
+) {
+  const newFiles = await db.insert(Schemas.files1).values(values).returning();
+  assert(newFiles.length == 1);
+  return newFiles[0];
+}
+
+export async function updateProfilePictureUrlToUser(
+  uploaderid: number,
+  values: { url: string },
+) {
+  const updatedUser = await db
+    .update(Schemas.files1)
+    .set(values)
+    .where(eq(Schemas.files1.uploaderid, uploaderid))
```

Comment:
this is a big big bug ... can you spot it?

Reply:
I will fix all of these bugs.